src/sestest/ServerCommandLine.cs

```diff
@@ -27,10 +27,10 @@ public sealed class ServerCommandLine : CommandLineBase
             Justification = "Must be a string to support our commandline.")]
         public string ServerUrl { get; set; }
 
-        [Option("audience", HelpText = "[Internal] Audience to which all tokens must be addressed (optional).")]
+        [Option("audience", HelpText = "[Internal] Audience to which all tokens must be addressed (optional, defaults to '" + Claims.DefaultAudience + "').")]
```

Comment:
Can you use `$"interpolated {strings}"` with attributes?

Might be nicer to write
```
[Option("audience", HelpText = $"[Internal] Audience to which all tokens must be addressed (optional, defaults to '{Claims.DefaultAudience}').")]
```

Reply:
Sadly not...